### PR TITLE
Proofread jwt

### PR DESCRIPTION
**Pull request by greg0ire**
The diff would be easier to read if you capped lines to some limit (120 chars?). Consider it.

### CODE REVIEW

**Comment by soyuka** [comment]
> The diff would be easier to read if you capped lines to some limit (120 chars?). Consider it.
It is/should be?

**Comment by greg0ire** [comment]
I'm not sure I get your question(s?)

> Is it capped to 120 chars

No, far from it, there does not seem to be a limit at all.

> It should be

If you want easier diffs, probably.

**Comment by greg0ire** [comment]
> If you want easier diffs, probably.

github helps a lot with the word highlight though

**Comment by soyuka** [comment]
> github helps a lot with the word highlight though

yes haha, and I thought we had a limit

**Comment by dunglas** [comment]
We have a limit, but we don't enforce it to not annoy the contributors... Maybe should we add a check for that and some guidance on how to fix it automatically.

File: core/jwt.md [hunk 1]
```diff
@@ -74,14 +74,14 @@ api_platform:
...
-And the Authorize button will automatically appear in Swagger UI.
+And the "Authorize" button will automatically appear in Swagger UI.
 
 ![Screenshot of API Platform with Authorize button](images/JWTAuthorizeButton.png)
 
 ### Adding a New API Key
 
-All you have to do is to configure the API key in the value field.
-By default [only the authorization header mode is enabled](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/index.md#2-use-the-token) in [LexikJWTAuthenticationBundle](https://github.com/lexik/LexikJWTAuthenticationBundle).
+All you have to do is configure the API key in the `value` field.
```
Comment (dunglas) [inline core/jwt.md]: configuring?

**Comment by dunglas** [comment]
Thanks @greg0ire

**Pull request merged**